source/server/active_raw_udp_listener_config.cc

```diff
@@ -24,7 +24,9 @@ ProtobufTypes::MessagePtr ActiveRawUdpListenerConfigFactory::createEmptyConfigPr
 
 Network::ActiveUdpListenerFactoryPtr
 ActiveRawUdpListenerConfigFactory::createActiveUdpListenerFactory(
-    const Protobuf::Message& /*message*/) {
+    const Protobuf::Message& message) {
+  auto& config =
```

Comment:
for now this cast will not work, need to reorg udp listener proto config to have one top level message